zaza/openstack/charm_tests/glance/setup.py

```diff
@@ -87,11 +93,20 @@ def add_lts_image(glance_client=None, image_name=None, release=None):
     :param release: Name of ubuntu release.
     :type release: str
     """
+    image_arch = os.environ.get('image_arch')
+    if not image_arch:
+        image_arch = "amd64"
+    logging.info("Image architecture set to {}".format(image_arch))
+    if image_arch == "arm64":
+        properties = {'hw_firmware_type': 'uefi'}
+    if image_arch == "ppc64el":
+        properties = {'architecture': 'ppc64'}
```

Comment:
imo it's neater to use a dict here rather than multiple `if` statements:

```
arch_image_properties = {
    'arm64': {'hw_firmware_type': 'uefi'},
    'ppc64el' {'architecture': 'ppc64'}}
properties = properties or arch_image_properties.get(image_arch)
```